lob.go

```diff
@@ -264,16 +264,38 @@ func (dl *DirectLob) ReadAt(p []byte, offset int64) (int, error) {
 // WriteAt writes p starting at offset.
 func (dl *DirectLob) WriteAt(p []byte, offset int64) (int, error) {
 	if !dl.opened {
-		//fmt.Printf("open %p\n", lob)
+		// fmt.Printf("open %p\n", lob)
 		if C.dpiLob_openResource(dl.dpiLob) == C.DPI_FAILURE {
 			return 0, errors.Errorf("openResources(%p): %w", dl.dpiLob, dl.conn.getError())
 		}
 		dl.opened = true
 	}
-
+	
 	n := C.uint64_t(len(p))
 	if C.dpiLob_writeBytes(dl.dpiLob, C.uint64_t(offset)+1, (*C.char)(unsafe.Pointer(&p[0])), n) == C.DPI_FAILURE {
 		return int(n), errors.Errorf("writeBytes: %w", dl.conn.getError())
 	}
 	return int(n), nil
 }
+
+// GetFileName Return []string consisting of the directory alias and file name for a BFILE type LOB.
+func (dl *DirectLob) GetFileName() ([]string, error) {
```

Comment:
Please let it return two (named) strings (directoryAlias, fileName string, err error)
or a struct (type DirAndFile struct {Dir, File string}), not a slice of strings.